uxarray/dataset.py

```diff
@@ -1,10 +1,10 @@
 """uxarray dataset module."""
 
 from .grid import *
-from pathlib import PurePath
+from .helpers import parse_grid_type
 
 
-def open_dataset(grid_filename, *args):
+def open_dataset(grid_filename, *args, **kw):
```

Comment:
Not for this PR but IMO we should consider deleting this.

There are ways to create an xarray Dataset of unstructured mesh data without going through `open_dataset` or `open_mfdataset`: for example, through `intake` or `fsspec`. 

And as Filipe had to thread through `**kw` here, simple `open_dataset` and `open_mfdataset` calls can fail (especially the latter!) without special kwargs.